.github/workflows/ci.yml

```diff
@@ -96,6 +100,24 @@ jobs:
           echo "LD_LIBRARY_PATH=$LLVM_PREFIX/lib:`dirname $ASAN_DSO`" >> $GITHUB_ENV
           echo "$LLVM_PREFIX/bin" >> $GITHUB_PATH
 
+      - name: Prepare TSan
+        if: matrix.build.sanitize == 'tsan'
+        env:
+          LLVM_PREFIX: /usr/lib/llvm-19
+        run: |
+          TSAN_DSO=`$CC -print-file-name=libclang_rt.tsan-x86_64.so`
+          echo "LDSHARED=$CC -shared" >> $GITHUB_ENV
+          echo "CPPFLAGS=-g -fsanitize=thread -fno-sanitize=function -fno-omit-frame-pointer -fopenmp -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION" >> $GITHUB_ENV
+          echo "LDFLAGS=-g -fsanitize=thread -shared-libasan -fopenmp=libomp" >> $GITHUB_ENV
```

Comment:
Linking fails without `-shared-libasan` (even for TSan). I'm not sure why that's the case